specs/engage-digital_openapi3.yaml

```diff
@@ -136,7 +136,7 @@ paths:
           name: topologyId
           required: true
           schema:
-            type: string
+            type: integer

```

Comment:
But it is a string, no?

Reply:
yes it was a test for something else, you should wait for the PR to be assigned and ready before reviewing ^^